cypress.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = defineConfig({
     setupNodeEvents(on, config) {
       on("task", {
         generateUser() {
-          let randomNumber = Math.ceil(Math.random(1000) * 1000);
+          let randomNumber = Math.ceil(Math.random(1000) * 1000000);
```

Comment:
Math.random() function doesn't take arguments. It returns a floating-point, pseudo-random number in the range 0 to less than 1. Use it without arguments to generate a random number.